packages/sdk-middleware-auth/test/anonymous-session-flow.spec.js

```diff
@@ -16,7 +16,7 @@ function createTestRequest(options) {
 
 function createTestMiddlewareOptions(options) {
   return {
-    host: 'https://auth.commercetools.co',
+    host: 'https://auth.europe-west1.gcp.commercetools.com',
```

Comment:
just as a heads up, technically these pointed towards US env but shouldn't matter for tests.